src/pages/FrontPage.js

```diff
@@ -9,14 +9,16 @@ import TeacherInfobox from '../components/FrontPage/TeacherInfobox';
 import WelcomeBox from '../components/FrontPage/WelcomeBox';
 
 export const  FrontPage = React.createClass({
-
+  componentDidMount() {
+    document.title = 'Kodeklubben';
```

Comment:
I stedet for å gjøre noen endringer i denne filen, så foreslår jeg at du heller endrer på `webpack.config.babel.js`. Der står det at title skal være "Kodeklubben (server)" for statiske sider omtrent på linje 112.